Docs/pagedeployment.md

```diff
@@ -0,0 +1,32 @@
+# ELEMWAVE PAGE DEPLOYMENT
+In order to deploy a public web site using GitHub pages we have to follow this steps:
+
+    1. First of all we have to create a repository where we will keep our web site configuration and programmation (It has to be public if you want to use the free version of Git pages).
```

Comment:
The list doesn't look right when visualising the file:
![image](https://github.com/elemwave/website/assets/105167930/67bfcddc-d5d9-4db4-b9b3-abb06f31cf3a)
That format could be appropriate for a fragment of code, for this a proper list would make it easier to read.